grants/faq.md

```diff
@@ -0,0 +1,153 @@
+</details>
+
+<details>
+  <summary><b>Can anyone apply?</b></summary>
+
+  Teams that have sold their own token are not eligible for a Web3 Foundation grant. Other than that, there are no restrictions.
+</details>
+
+<details>
+  <summary><b>My application was rejected. Do you have any recommendations on where to go from here?</b></summary>
+
+  We usually give reasons why an application was rejected. We always try to be constructive and work with you towards an application that is beneficial to all parties. If we find no common ground, please have a look at [this section in our General Grants readme](https://github.com/w3f/General-Grants-Program#rocket-alternative-funding-sources) for a list of alternative funding opportunities.
+</details>
+
+<details>
+  <summary><b>One of your grantees is using my code without respecting the terms of its license.</b></summary>
+
+  Please [reach out to us](mailto:grants@web3.foundation) asap.
+</details>
+
+<details>
+  <summary><b>Something came up and I cannot finish the project in time. Can we postpone or call off the rest of my project?</b></summary>
+
+  The Web3 Foundation reserves the right to terminate an agreement that is behind schedule. However, we are not interested in taking away your grant for any slight hiccup. More often than not, delays are part of the journey and do not constitute a reason for concern. The best way to handle changes in your plans is to get in touch with us. If you like to prematurely end your work, we can amend your application and remove the milestones you won't be able to complete. If you decide to continue work at a later date, you can always reapply for the remaining milestones and potentially adapt them to take into account any insights you have gained in the meantime.
+</details>
+
+<details>
+  <summary><b>Why are there grant applications being accepted or rejected that were submitted after mine?</b></summary>
+
+  There are many reasons why your application might take longer than others: some applications are straightforward and simple and address an obvious issue, others require deeper understanding and discussion. If your application is highly technical or specialised, we might have to bring in an external evaluator. Sometimes, this specialised evaluator is busy with another evaluation. And sometimes, the committee is simply unsure or not quite convinced.
+</details>
+
+
+## 🔓 Open Grants
+
+<details>
+  <summary><b>When can I apply for a follow-up grant?</b></summary>
+
+  Anyone who has successfully completed a grant project (all milestones were accepted, or the previous grant was nullified in mutual agreement) can apply for a follow-up grant.
```

Comment:
I would opt for the same terminology as elsewhere since we don't use `nullify` anywhere else and this might confuse people. I think the "in mutual agreement" is clear enough. Otherwise I also prefer `cancelled`

```suggestion
  Anyone who has successfully completed a grant project (i.e. all milestones were accepted, or the previous grant was terminated in mutual agreement) can apply for a follow-up grant.
```